src/data-sources/data-source-list.scss

```diff
@@ -0,0 +1,125 @@
+.data-source-list-card {
+
+	.add-data-source-btn {
+		border-radius: 4px;
+		padding-right: 8px;
+
+		svg {
+			margin-left: 4px;
+		}
+	}
+}
+
+.settings_page_remote-data-blocks-settings {
+
+	.add-data-source-dropdown-content {
+		top: -7px !important;
```

Comment:
Wondering if there's a way to get this done without `!important` by being increasing specificity of the style as that should be used as last resort. I guess it's fine if this is the only way.

Reply:
Done in https://github.com/Automattic/remote-data-blocks/pull/65/commits/d73c11b3c8c40aff943a4dd3bf390a16482c8bad

Reply:
This styling was applied via the `style` attribute so `!important` seems like the only way.